parser-wrapper/parser-interface/src/main/java/org/mule/raml/interfaces/model/parameter/IParameter.java

```diff
@@ -16,5 +18,8 @@
     String message(String value);
     String getDisplayName();
     String getDescription();
+    String getExample();
+    String getExampleName();
```

Comment:
example name is always `null` when using the key `example` so `getExampleName()` does not make much sense, if you're OK I can remove this method an merge the rest of the changes